src/NSwag.CodeGeneration.CSharp/SwaggerToCSharpControllerGeneratorSettings.cs

```diff
@@ -24,6 +24,8 @@ public SwaggerToCSharpControllerGeneratorSettings()
             UseCancellationToken = false;
             AspNetNamespace = "System.Web.Http";
             RouteNamingStrategy = CSharpControllerRouteNamingStrategy.None;
+            UseModelValidationAttributes = false;
+            RequiredAttributeType = "System.ComponentModel.DataAnnotations.Required";
```

Comment:
ASP.NET Core needs the BindRequiredAttribute to make parameters required not the RequiredAttribute

Reply:
I just read a post which may be useful here: [have BindAttribute behave as BindRequiredAttribute in ASP.NET Core MVC](https://www.strathweb.com/2017/12/required-and-bindrequired-in-asp-net-core-mvc) - see _"But I don’t want to mix BindRequiredAttribute and RequiredAttribute"_